test/importing.coffee

```diff
@@ -2,20 +2,6 @@
 # ---------
 
 unless window? or testingBrowser?
-  test "coffeescript modules can be imported and executed", ->
```

Comment:
I don't see the point of this test. If a commonJS module is being imported then it is also being executed. Should we just leave this test out for good?